backend/contributions/admin.py

```diff
@@ -33,12 +33,20 @@ class GlobalLeaderboardMultiplierInline(admin.TabularInline):
 
 @admin.register(ContributionType)
 class ContributionTypeAdmin(admin.ModelAdmin):
-    list_display = ('name', 'category', 'is_default', 'is_submittable', 'get_current_multiplier', 'min_points', 'max_points', 'description', 'created_at')
+    list_display = (
+        'name', 'category', 'is_default', 'is_submittable', 'get_current_multiplier',
+        'min_points', 'max_points', 'icon', 'description', 'examples', 'created_at'
+    )
     list_display_links = ('get_current_multiplier',)
-    list_editable = ('name', 'is_default', 'is_submittable', 'description')
+    # Enable quick edits for common fields directly from the list view
+    list_editable = (
+        'name', 'category', 'is_default', 'is_submittable', 'min_points', 'max_points', 'icon', 'description', 'examples'
```

Comment:
This is A LOT. and examples is JSON, that's a no-no on the list_editable.. let's go back to what we had before.